internal/configuration/validator/const.go

```diff
@@ -66,16 +72,15 @@ var SecretNames = map[string]string{
 	"OpenIDConnectIssuerPrivateKey": "identity_providers.oidc.issuer_private_key",
 }
 
-// validKeys is a list of valid keys that are not secret names. For the sake of consistency please place any secret in
+var reKeyReplacer = regexp.MustCompile(`\[\d+]`)
```

Comment:
If the opening bracket is escaped, why should not the closing one be too?

Reply:
It's not required with go's regex parser, if there is no opening bracket closing brackets are treated as literals.

Reply:
Ok, did not know that.